download/install.js

```diff
@@ -13,4 +13,6 @@ if (fs.existsSync(src)) {
   process.exit(0)
 }
 
+// Hack to force chalk display colors
+process.argv.push('--color');
```

Comment:
Colors are needed for red warning https://github.com/zeit/now-cli/blob/eacces-rerun-with-unsafe-perm/download/src/log.js#L24. You are right in general case but if you try to run your process.env code, it does not work.